server/server/src/file_service.rs

```diff
@@ -216,8 +216,9 @@ pub async fn change_doc(
             return Err(ClientError(DocumentDeleted));
         }
 
+        tx.last_seen.insert(owner, get_time().0 as u64);
         // Here is where you would check if the person is out of space as a result of the new file.
-        // You could make this a transaction and check whether or not this is an increase in size or
+        // You could make this a transaction and che™ck whether or not this is an increase in size or
```

Comment:
?

Reply:
Lockbook™ 
lol, but that was a typo, apologies. As a windows user, I have the reflex of hitting ctrl+ 2 to switch tabs in vscode. In mac, I sometimes hit option+2 due to that reflex. option + 2 = ™